controllers/pipeline/pipeline_adapter.go

```diff
@@ -476,6 +476,14 @@ func (a *Adapter) prepareSnapshot(application *applicationapiv1alpha1.Applicatio
 		// We omit this not-yet-built component from the snapshot rather than
 		// including a component that is incomplete.
 		if containerImage == "" {
+			logger.Error(nil, "component cannot be added to snapshot for application due to missing containerImage", "component.Name", applicationComponent.Name, "application.Name", application.Name)
```

Comment:
I think we don't need to log the `application.Name` since #164 now automatically adds it to logs.

Reply:
Yes, thanks. But I need to add one line `logger := a.logger.WithApp(*application)` before calling `logger.Error()`.
Updated.